examples/combined-odata-mock/varkes_config.js

```diff
@@ -4,7 +4,9 @@ module.exports = {
         {
             name: "courses",
             metadata: "/courses/metadata",
-            specification_file: "apis/courses.xml"
+            specification_file: "apis/courses.xml",
+            oauth: "/authorizationserver/oauth/token",
+            apiType: "odata"

```

Comment:
one last remark: please rename the attribute to "type" only. It used only in an "apis" array, so having a prefix "api" is redundant. I will approve already so that you can merge immediately after the change.